script.js

```diff
@@ -1,4 +1,68 @@
+// initialise first mode asking input
```

Comment:
From the first second- great code organisation! Clear sections with comments above each section. Very easy to read. Good job.